features/searchgov_search.feature

```diff
@@ -38,6 +39,8 @@ Feature: SearchGov search
     When I am on epa's search page
     And I search for "exciting"
     Then I should see "exciting news"
+    When I am on epa's "Press" mobile news search page
+    Then I should see "Refine your search"
```

Comment:
This ensures that we continue to display the date refiners on the news SERP, because news items have consistent date info.